Documentation/Upgrade/rook-upgrade.md

```diff
@@ -49,6 +49,9 @@ those releases.
 ## Breaking changes in v1.12
 
 * The minimum supported version of Kubernetes is v1.22.
+* CephCSI CephFS driver introduced a breaking change in v3.9.0. Please follow the steps mentioned in the
```

Comment:
Can you summarize the breaking change in one or two sentences here? In this Rook upgrade guide, they should understand if they are affected, and if it's common. By default, users won't be affected, right? I don't expect it's common to use the mount options. For example, say that if they are setting the mountOptions in a CephFS storage class, they need to follow those instructions in the CSI upgrade guide.

Reply:
:+1: yes
Updated mentioning the above suggested details, 
Ptal